test_start.py

```diff
@@ -1,10 +1,25 @@
 from user import User
 from board import Board
-from start import start_game     
+from start import start_game 
 
 def test_prompt_game_notifies_user_of_a_draw():
     game_board = Board()
-    game_board.board_state = ["X", "O", "X", "O", "X", "O", "X", "O", "X"]
+    game_board.board_state = [
+                                "X", "O", "X", 
+                                "O", "X", "O", 
+                                "X", "O", "X"
+                             ]
     player1 = User("X")
     player2 = User("O")
-    assert start_game(game_board.board_state, player1.symbol, player2.symbol) == 'DRAW. Game Over';
+    assert start_game(game_board.board_state, player1.symbol, player2.symbol) == True 
+
+def test_prompt_game_notifies_user_of_a_win():
+    game_board = Board()
+    game_board.board_state = [
+                                "X", "X", "X", 
+                                " ", "X", "O", 
+                                "X", "O", "X"
+                             ]
+    player1 = User("X")
+    player2 = User("O")
+    assert start_game(game_board.board_state, player1.symbol, player2.symbol) == True 
```

Comment:
Still quite a bit of work to do here. You're assertions, don't really mean anything. There's no visibility into why the `start_game` function returns a boolean or what the state of the board is when it does. For all I know it could return true for any board state.

Reply:
Yeah start_game is pretty rough. I attempted to plug in everything and use start_game as a gigantic loop.